velox/connectors/hive/storage_adapters/hdfs/HdfsWriteFile.cpp

```diff
@@ -49,6 +49,11 @@ void HdfsWriteFile::close() {
 }
 
 void HdfsWriteFile::flush() {
+  // Check Whether the file is opened for write.
+  VELOX_CHECK_EQ(
+      hdfsFileIsOpenForWrite(hdfsFile_),
+      1,
+      "Flush failure in HdfsWriteFile::flush as the file is not opened");
```

Comment:
> Flush failure in HdfsWriteFile::flush

"Flush" repeats unnecessarily. Also, let's add file path to the error message to help troubleshooting.

"Cannot flush HDFS file because file is not opened for write: {}", filePath_